src/data/posts/twitter-and-the-perils-of-obedience.mdx

```diff
@@ -0,0 +1,164 @@
+
+I can't help but draw a connection between the Milgram experiment and the current state of Twitter. The experiment was designed to test the willingness of people to obey authority figures. Twitter's recent acquisition by a megalomaniacal celebrity narcissist is an amplified example of this phenomenon in action.
+
+Under Musk's leadership, Twitter's employee base has been slashed from 7,500 to approximately 2,000 in the last month, in an attempt to cut costs after a purchase that only made sense for a rich, vein billionaire who wanted to flex his ego.
+
+I've seen plenty of discussion online that many of twitter's remaining engineers are likely tied to H1B visas, and are unable to leave their job without risking deportation. There is no doubt another contingent of employees who believe in Musk's vision, who are bought into the idea of being an elite engineer working on a desperately slim team, and are willing to work for him despite the fact that he's a complete and utter asshole.
+
+I don't think it is a stretch to say that employees at Twitter are being treated poorly, and have been given increasingly shocking orders from their new CEO (for example, [1](https://twitter.com/ZoeSchiffer/status/1593649356661436417), [2](https://twitter.com/alexeheath/status/1595701492006014977?t=7MMiqyX0-LWIXExz6QtkAQ&s=19)).
+
+Since rumors of Musk's takeover, legions of advertisers have pulled their ad spend from Twitter, for fear of being associated with the company. The company's income has plummeted, and Musk is clearly trying to cut costs wherever he can, while adding features as a part of a desperate money grab to keep the company from going bankrupt.
+
+I left twitter recently, shortly after a hasty user poll resulted in the reinstatement of former (and twice-impeached) president Trump's account.
+
+<Tweet align="center" tweetLink="irreverentmike/status/1594455660040572928" />
+
+Since then, another hasty poll has resulted in an absolutely staggering policy change: all\* accounts that were previously suspended for violating Twitter's rules will be reinstated.
+
+<Tweet align="center" tweetLink="elonmusk/status/1595473875847942146" />
+
+Pandora's box has been opened, and the monsters are coming back. Twitter is now a platform for hate speech, misinformation, and disinformation. It's will become a cesspool of conspiracy theories, and a breeding ground for countless nightmares and real-world dangers.
+
+This is a very real, very observable truth: countless racists, abusers, and other dangerous people are being given "amnesty" to come back the platform.
+
+I've seen many others abandon their twitter accounts in the wake of these changes, but -- friends -- I'm honestly shocked that so many people choose to remain.
+
+Why are so many people still on twitter, administer electric shocks to the collective public by keeping their accounts alive?
+
+I'm reminded by the words of Stanley Milgram, in his reflection on the results of his now-infamous experiment:
```

Comment:
Stanford prison experiments also come to mind. Where participants were so consumed with their assumed role they lost morality.